Dockerfile

```diff
@@ -176,7 +176,7 @@ RUN if [[ ${AIRFLOW_PRE_CACHED_PIP_PACKAGES} == "true" ]]; then \
        fi; \
        pip install --user \
           "https://github.com/${AIRFLOW_REPO}/archive/${AIRFLOW_BRANCH}.tar.gz#egg=apache-airflow[${AIRFLOW_EXTRAS}]" \
-          --constraint "https://raw.githubusercontent.com/apache/airflow/${AIRFLOW_CONSTRAINTS_REFERENCE}/constraints-${PYTHON_MAJOR_MINOR_VERSION}.txt" \
+          --constraint "${AIRFLOW_CONSTRAINTS_LOCATION}" \
```

Comment:
The same URL is defined at 

https://github.com/apache/airflow/blob/780c0b4ab8d1a64ca806942ae094f722d9beb447/Dockerfile#L158-L159